modules/p2/ding_list/theme/theme.inc

```diff
@@ -308,5 +342,17 @@ function template_preprocess_ding_list_add_button(&$variables) {
   else {
     $variables['classes_array'][] = 'ding-list-add-button--multiple-actions';
   }
+}
+
+/**
+ * Ding list label/value theme.
+ */
+function theme_ding_list_label_value($variables) {
+  $label = '';
+  if ($variables['label']) {
+    $label = '<div class="label">' . $variables['label'] . '</div>';
+  }
+  $out = '<div class="field">' . $label . '<div class="text">' . $variables['value'] . '</div></div>';
```

Comment:
These class names are very generic. If this is to be considered a component (it has a separate theme function so I think that is a reasonable assumption) then I would expect it to be namespaced as such e.g `ding-list-field`, `ding-list-field-label` and `ding-list-field-text`.

Reply:
Is changed